system-tests/tests/smoke/cre/por_test.go

```diff
@@ -813,8 +833,10 @@ func TestCRE_OCR3_PoR_Workflow_SingleDon_MockedPrice(t *testing.T) {
 	validateEnvVars(t, in)
 	require.Len(t, in.NodeSets, 1, "expected 1 node set in the test config")
 
-	binaryDownloadOutput, err := downloadBinaryFiles(in)
-	require.NoError(t, err, "failed to download binary files")
+	if os.Getenv("CI") == "true" {
```

Comment:
Do we set this flag in CI or is it a default GHA flag? @Tofel

Reply:
we set it